mm2src/mm2_net/src/wasm/wasm_ws.rs

```diff
@@ -50,6 +50,15 @@ pub enum InitWsError {
 impl InitWsError {
     fn from_ws_new_err(e: JsValue, url: &str) -> InitWsError {
         let reason = stringify_js_error(&e);
+
+        // Check for TypeError
+        if reason.contains("URL constructor") {
+            return InitWsError::InvalidUrl {
+                url: url.to_owned(),
+                reason,
+            };
+        };
+
         match e.dyn_ref::<DomException>().map(DomException::code) {
```

Comment:
`DomException::code` is is no longer recommended btw, ref. https://developer.mozilla.org/en-US/docs/Web/API/DOMException/code we should use [`DomException::name`](https://developer.mozilla.org/en-US/docs/Web/API/DOMException/name). This is not related to this PR but maybe we should do it in this PR if it's not too complicated. Btw, do we need special handling of `URL constructor` if we did this?

Reply:
thanks. done

Reply:
> Btw, do we need special handling of URL constructor if we did this?

not really if the error includes `URL constructor` then it's a url parsing error not need to overkill